Thumbprint/Targets/Playground/Classes/Inspectable/InspectableViews/IconButton+InspectableView.swift

```diff
@@ -18,7 +18,7 @@ extension IconButton: InspectableView {
     }
 
     static func makeInspectable() -> UIView & InspectableView {
-        let button = IconButton(icon: Icon.contentActionsFilterMedium, accessibilityLabel: "Demo icon button")
+        let button = IconButton(icon: Icon.contentActionsFilterMedium.image, accessibilityLabel: "Demo icon button")
```

Comment:
not neccesarily something you need to address as it's a personal pet peeve, but it bothers me when things have a parameter called "icon" but the then they don't take an "icon" but an "image" 😬